addons/l10n_hu_edi/models/account_move.py

```diff
@@ -339,10 +339,15 @@ def _l10n_hu_edi_check_invoices(self):
                 'message': _('Please set company Country, Zip, City and Street!'),
                 'action_text': _('View Company/ies'),
             },
-            'company_not_huf': {
-                'records': self.company_id.filtered(lambda c: c.currency_id.name != 'HUF'),
-                'message': _('Please use HUF as company currency!'),
-                'action_text': _('View Company/ies'),
```

Comment:
Do we have to remove this because now the company are now not guaranteed to be hungarian? How about we keep it but add another check to see if the company is from Hungary?
```py
'records': self.company_id.filtered(lambda c: c.country_code == 'HU' and c.currency_id.name != 'HUF'),
```